website/docs/d/clusters.html.markdown

```diff
@@ -23,9 +23,15 @@ resource "mongodbatlas_cluster" "test" {
   project_id   = "<YOUR-PROJECT-ID>"
   name         = "cluster-test"
   disk_size_gb = 100
-  num_shards   = 1
-
-  replication_factor           = 3
+  replication_specs {
```

Comment:
This change won't work with cluster_type (in the api clusterType) so that needs to be added for both datasource and resource examples.  Always test changes - I tried this and it failed immediately.

Reply:
fixed